app/views/catalog/_form_server_profile_template_options.html.haml

```diff
@@ -0,0 +1,15 @@
+- url = url_for(:id => "#{@edit[:rec_id] || "new"}", :action => "atomic_form_field_changed")
+
+#options_div
+  .form-horizontal
+    - opts = [["<#{_('Choose')}>", nil]] + @edit[:available_server_profile_templates]
+    .form-group
+      %label.col-md-2.control-label
+        = _('Server Profile Template')
+      .col-md-8
+        = select_tag('server_profile_template_id',
+                          options_for_select(opts, @edit[:new][:server_profile_template_id]),
+                          "data-miq_sparkle_on" => true,
+                          :class                => "selectpicker")
+        :javascript
+              miqSelectPickerEvent('server_profile_template_id', '#{url}')Ï
```

Comment:
there is special character at the end

Reply:
an extra space in here as well, can be fixed like
<img width="602" alt="image" src="https://user-images.githubusercontent.com/87487049/209546309-91849cf1-5fc9-4faa-b2f5-90269737c351.png">